src/cms/models/pages/__init__.py

```diff
@@ -1,5 +1,8 @@
 """
-This package contains both page-related data models:
+This package contains page-related data models:
+:class:`~cms.models.pages.abstract_base_page.AbstractBasePage` and
 :class:`~cms.models.pages.page.Page` and
 :class:`~cms.models.pages.page_translation.PageTranslation`
+:class:`~cms.models.pages.imprint_page.ImprintPage` and
+:class:`~cms.models.pages.imprint_page_translation.ImprintPageTranslation` and
```

Comment:
Sorry, didn't notice that `AbstractBasePageTranslation` was also missing and the conjunctions are inconsistent. I propose a bullet list:
```suggestion
This package contains page-related data models:

* :class:`~cms.models.pages.abstract_base_page.AbstractBasePage` and
  :class:`~cms.models.pages.abstract_base_page_translation.AbstractBasePageTranslation`
* :class:`~cms.models.pages.page.Page` and
  :class:`~cms.models.pages.page_translation.PageTranslation`
* :class:`~cms.models.pages.imprint_page.ImprintPage` and
  :class:`~cms.models.pages.imprint_page_translation.ImprintPageTranslation`
```

You can test the resulting docs by running `./dev-tools/generate_documentation.sh` and open `./docs/autogenerated/cms.models.pages.html` in your browser afterwards.